packages/wallets/src/components/DerivAppsSection/DerivAppsSection.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useActiveWalletAccount } from '@deriv/api';
+import { DerivAppsGetAccount } from './DerivAppsGetAccount';
+import { DerivAppsTradingAccount } from './DerivAppsTradingAccount';
+import './DerivAppsSection.scss';
+
+const DerivAppsSection: React.FC = () => {
+    const { data: activeWallet } = useActiveWalletAccount();
+    const tradingAccountLoginId = activeWallet?.linked_to?.find(account => account?.platform === 'dtrade')?.loginid;
+
+    return tradingAccountLoginId ? (
+        <DerivAppsTradingAccount
+            isDemo={activeWallet?.is_virtual}
+            label={activeWallet?.landing_company_name}
+            tradingAccountLoginId={tradingAccountLoginId}
```

Comment:
Since there are no mapping happening here, can we just use these values in this component itself by calling `useActiveWalletAccount` there ?

Reply:
@thisyahlen-deriv It's because I already did this in `DerivAppsSection` component and `DerivAppsTradingAccount` and `DerivAppsGetAccount` components a just dummy components which can be used just inside of the `DerivAppsSection` component. Also I use array method `find` in `DerivAppsSection` to find trading account loginid so I don't see any sense to do the same inside of the `DerivAppsTradingAccount`